queries.sql

```diff
@@ -3,7 +3,7 @@ SELECT * FROM animals WHERE name LIKE '%mon%';
 SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth < '2019/12/30';
 SELECT name FROM animals WHERE neutered = 't' AND  escape_attempts < 3;
 SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
-SELECT name, escape_attempts FROM animals WHERE weight_kg < 10.5;
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 't';
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
## NOTICE
This change applies to the `data.sql` file as I can see you've worked on it before but you're missing some requirements. Kindly check below:

- [ ]  Please, update the following lines of code to meet the [Vet clinic database: create animals table](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md) milestone requirement.
- [ ] Kindly make the value of the weight_kg a floating number and not an integer. e.g from `8` to `8.0`. Do this to the rest that is not floating.

   - /* Populate database with sample data. */
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon',  TO_DATE('03/02/2020', 'DD/MM/YYYY'), 0, TRUE, 10.23 );
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', TO_DATE('15/11/2018', 'DD/MM/YYYY'), 2, TRUE, 8);
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', TO_DATE('07/01/2021', 'DD/MM/YYYY'), 1, FALSE, 15.04);
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', TO_DATE('12/05/2017', 'DD/MM/YYYY'), 5, TRUE, 11 );

[OPTIONAL] :point_right: *I think using the TO_DATE() is making you write extra lines of code because, [you're not using it correctly](https://www.postgresqltutorial.com/postgresql-date-functions/postgresql-to_date/). why still specify the format of the date when you're already using a date format? Kindly read more about it from the link*



---

Reply:
- [ ] Please, let's also kindly make the value of the weight_kg a floating number and not an integer. e.g 
` '-11.0', '-45.0', '17.0', '22.0'` in these [lines of code](https://github.com/t-bello7/micro-vet-clinic-database/blob/7936a02af04f466e13d1d8dd69b5febdbce1ea24/data.sql#L9-L17)










```